### DISCUSSION TITLE
Allow folders to work with cm_MarkCurrentExtension

### DISCUSSION THREAD

**Discussion by advice2020** [doublecmd/doublecmd#1405] [Suggestions]
Not sure if this is possible in Linux but would be great if could be able to use cm_MarkCurrentExtension & cm_UnmarkCurrentExtension commands with folders.  So running this command on folder would select or deselect only all folders in current directory.  Currently running this command on folders just selects all files.

This would function similar to the following which marks all directories
cm_markmarkall
attr=d+

Thank You

**Comment by Skif-off** [comment]
This seems counterintuitive: filename extension is intended to identify the file type/format.

(Yes, of course, a folder is a special file, but a specific attribute makes this object a separate entity.)

**Comment by advice2020** [comment]
@Skif-off
You are everywhere :)
Thank you for your response. 
I don't know what "specific attribute makes this object a separate entity" means :) but I know, this might not fit exactly into the tools specific purpose, but I was looking at it more like this would allow this tool to cover both file types as well as folders eliminating the need for a separate hotkey that only does one thing, just selects folders, if that makes sense.
You know how everyone in Linux always says "in Linux everything is a file" :)
